### ISSUE TITLE
"make build" failed on go 1.13

### ISSUE THREAD

**Issue by hankei6km** [Songmu/gocredits#7]
## Background

I want to use `gocredits.Run()` in my code on go 1.13.

## Steps to reproduce

```console
$ docker run --rm -it golang:1.13.0 bash
root@71eb884d4063:/go# git clone https://github.com/Songmu/gocredits.git && cd gocredits
Cloning into 'gocredits'...
remote: Enumerating objects: 82, done.
remote: Counting objects: 100% (82/82), done.
remote: Compressing objects: 100% (61/61), done.
remote: Total 82 (delta 31), reused 61 (delta 16), pack-reused 0
Unpacking objects: 100% (82/82), done.
root@71eb884d4063:/go/gocredits# make build
go get  -d
go: downloading golang.org/x/xerrors v0.0.0-20190410155217-1f06c39b4373
go: extracting golang.org/x/xerrors v0.0.0-20190410155217-1f06c39b4373
go build -ldflags="-s -w -X github.com/Songmu/gocredits.revision=1b4cddf" ./cmd/gocredits
go: finding golang.org/x/xerrors v0.0.0-20190410155217-1f06c39b4373
# golang.org/x/xerrors
../pkg/mod/golang.org/x/xerrors@v0.0.0-20190410155217-1f06c39b4373/adaptor_go1_13.go:16:14: undefined: errors.Frame
../pkg/mod/golang.org/x/xerrors@v0.0.0-20190410155217-1f06c39b4373/format_go1_13.go:12:18: undefined: errors.Formatter
make: *** [Makefile:39: build] Error 2
```

## Workaround

```console
root@71eb884d4063:/go/gocredits# go mod edit -require=golang.org/x/xerrors@v0.0.0-20190717185122-a985d3407aa7
root@71eb884d4063:/go/gocredits# go mod tidy
go: downloading golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7
go: extracting golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7
root@71eb884d4063:/go/gocredits# make build
go get  -d
go build -ldflags="-s -w -X github.com/Songmu/gocredits.revision=1b4cddf" ./cmd/gocredits
go: finding golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7
root@71eb884d4063:/go/gocredits# ./gocredits -version
gocredits v0.0.6 (rev:1b4cddf)
```

**Comment by Songmu** [comment]
Thank you for reporting. I adjusted dependency at #8 and released as v0.0.7. Please try it.

Thanks!

**Comment by hankei6km** [comment]
Thank you for quick response.

I can use `gocredits.Run()` in my code on go 1.13.
https://github.com/hankei6km/go-ac/commit/4e018f3a50055be4969499d7d3a4b398ce1067e2

it simplified my `magefile.go` :-).
